java/squeek/veganoption/integration/waila/ProviderJutePlant.java

```diff
@@ -34,9 +35,9 @@ public ItemStack getWailaStack(IWailaDataAccessor accessor, IWailaConfigHandler
 		{
 			float growthValue = ((BlockJutePlant) accessor.getBlock()).getGrowthPercent(accessor.getWorld(), accessor.getPosition(), accessor.getBlockState()) * 100.0F;
 			if (growthValue < 100)
-				toolTip.add(String.format("%s : %.0f %%", I18n.format("hud.msg.growth"), growthValue));
+				toolTip.add(LangHelper.translate("waila.growth", (int) growthValue));
```

Comment:
On second thought, I think avoiding Math.round is being overly cautious. In the StackOverflow link, you'll notice that the 'bad performance' in Java 6 is still just 0.022ms for every 1000 iterations (seems negligible for our case), and the percentage of people actually running JRE 6 is probably astronomically low.